libp2p/RLPXFrameCoder.cpp

```diff
@@ -23,42 +23,46 @@
 
 #include <libdevcore/Assertions.h>
 #include "RLPxHandshake.h"
+#include "RLPXPacket.h"
```

Comment:
Looks like there is some inconsistency in the namings of the files. Prefixes RLPx... and RLPX...